### ISSUE TITLE
indent: line after annotation is falsely indented

### ISSUE THREAD

**Issue by jonnytest1** [typescript-eslint/typescript-eslint#4434]
- [x ] I have tried restarting my IDE and the issue persists.
- [x ] I have updated to the latest version of the packages.
- [x ] I have [read the FAQ](https://typescript-eslint.io/docs/linting/troubleshooting) and my problem is not listed.

**Repro**

```JSON
{
  "rules": {
    "@typescript-eslint/indent": ["warn",2, {
            "MemberExpression": 1,
            "FunctionDeclaration": {
              "body": 1,
              "parameters": "first" 
            },
            "CallExpression": {
              "arguments": "first"
            },
            "ArrayExpression": 1,
            "ImportDeclaration": 1,
            "SwitchCase": 1
          }]
  }
}
```

```TS
  @ViewChild('child', { static: true })
  child: ElementRef;
```

export class SomeClass {
  @ViewChild('child', { static: true })
  child: ElementRef;
}

**Actual Result**
export class SomeClass {
   @ViewChild('child', { static: true })
      child: ElementRef;                 // this line is indented (or a warning is thrown that the line should be)
}

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `5.9.1` |
| `@typescript-eslint/parser`        | 5.9.1` |
| `TypeScript`                       | `4.5.4` |
| `ESLint`                           | `8.0.0` |
| `node`                             | 16` |

**Comment by bradzacher** [comment]
#1824 - please use the search next time.
